### ISSUE TITLE
"close all" / "close others" options on tabs

### ISSUE THREAD

**Issue by jrlacy** [wkeese/old-api-viewer#41]
You can wind up with a lot of open tabs.  I'd like the ability to close them all without refreshing the page.  Basically just want to add 'close all' and 'close others' to the right-click tax context menu you get on the tabs.

**Comment by wkeese** [comment]
Yup, I've thought about this too, it would be a good option for the TabContainer in general.
